src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -183,6 +183,14 @@ public function theExchangeRateForWasChangedTo(CurrencyInterface $currency, $exc
     }
 
     /**
+     * @When /^the (currency "[^"]+") gets deleted$/
+     */
+    public function theCurrencyGetsDeleted(CurrencyInterface $currency)
+    {
+        $this->currencyRepository->remove($currency);
```

Comment:
The only problem being, there is no UI for deleting currencies. Guessing now it will be added with all the rework with them, but for now the options are: either set the scenario to `@todo` and revert the removing currency logic; leave it as is for now